ui/StatusQ/src/StatusQ/Controls/StatusInput.qml

```diff
@@ -414,10 +415,17 @@ Control {
         readonly property int inputHeight: statusBaseInput.multiline || (root.minimumHeight > 0) || (root.maximumHeight > 0)?
                                   Math.min(Math.max(statusBaseInput.topPadding + statusBaseInput.bottomPadding, 44,
                                                     root.minimumHeight), root.maximumHeight) : 44
+        readonly property real noPadding: -0.1
```

Comment:
Could you explain how it works and why there is this -0.1 magic value?

Reply:
I'm afraid it may introduce some sub-pixel alignments and visual glitches in some envs/platforms.